rotkehlchen/api/rest.py

```diff
@@ -3947,9 +3947,9 @@ def _query_kraken_staking_events(
 
         if exchanges_list is None:
             return {
```

Comment:
Both for this and below we have a function to do this dict wrapping: `wrap_in_fail_result()`